ql/src/test/results/clientpositive/perf/tez/constraints/query1b.q.out

```diff
@@ -210,7 +210,7 @@ STAGE PLANS:
                             Statistics: Num rows: 16855704 Data size: 2008197920 Basic stats: COMPLETE Column stats: COMPLETE
                             value expressions: _col2 (type: decimal(17,2))
                   Filter Operator
-                    predicate: (sr_store_sk is not null and sr_returned_date_sk is not null and sr_store_sk BETWEEN DynamicValue(RS_40_store_s_store_sk_min) AND DynamicValue(RS_40_store_s_store_sk_max) and in_bloom_filter(sr_store_sk, DynamicValue(RS_40_store_s_store_sk_bloom_filter))) (type: boolean)
+                    predicate: (sr_store_sk is not null and sr_returned_date_sk is not null) (type: boolean)
```

Comment:
SJ went away? It is kept in the other branch (L182).

Reply:
changes are gone in this file